tsconfig.json

```diff
@@ -42,9 +44,9 @@
     /* Module Resolution Options */
     "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
     "baseUrl": "./" /* Base directory to resolve non-absolute module names. */,
-    "paths": {
-      "*": ["*", "types/*"]
-    } /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */,
+    // "paths": {
```

Comment:
this is unnecessary because redux is a top-level dep, we don't import any other types